modules/rostests/winetests/wtsapi32/CMakeLists.txt

```
@@ -1,6 +1,7 @@
 
+add_definitions(-D__WINESRC__)
 add_definitions(-DUSE_WINE_TODOS)
 add_executable(wtsapi32_winetest wtsapi.c testlist.c)
 set_module_type(wtsapi32_winetest win32cui)
-add_importlibs(wtsapi32_winetest wtsapi32 msvcrt kernel32)
+add_importlibs(wtsapi32_winetest wtsapi32 advapi32 ntdll msvcrt kernel32)
```

```suggestion
add_importlibs(wtsapi32_winetest wtsapi32 advapi32 msvcrt kernel32 ntdll)
```
always terminate with ntdll at the end (if ntdll is needed).